lib/code_corps/stripe/adapters/stripe_card.ex

```diff
@@ -0,0 +1,9 @@
+defmodule CodeCorps.Stripe.Adapters.StripeCard do
+  import CodeCorps.MapUtils, only: [rename: 3]
+
+  def params_from_stripe(%{} = stripe_map) do
+    stripe_map
+    |> rename("id", "id_from_stripe")
+    |> rename("customer", "stripe_customer_id")
```

Comment:
It looks like we may want this to be the `customer_id_from_stripe`, if we want that at all.